src/protocol-only/notebookConcatDocument.ts

```diff
@@ -2,8 +2,8 @@
 // Licensed under the MIT License.
 
 'use strict';
-import * as vscode from 'vscode';
-import * as protocol from 'vscode-languageclient/node';
+import * as vscodeUri from 'vscode-uri';
+import * as protocol from 'vscode-languageclient';
```

Comment:
Not sure if it helps but we get all the types for TextDocument, Range, Position from `vscode-languageserver-types` lib

Reply:
Oh thanks. I'll try that and see what happens. Should be the same I think.

Reply:
Didn't work, but 'vscode-languageserver-protocol' seems to work. I'll switch to that one.